examples/scale/tickets/playlistticket.go

```diff
@@ -0,0 +1,125 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tickets
+
+import (
+	"math/rand"
+	"time"
+
+	"open-match.dev/open-match/pkg/pb"
+)
+
+var (
+	regions = []string{
+		"region.europe0",
+		"region.europe1",
+		"region.europe2",
+		"region.europe3",
+		"region.europe4",
+		"region.europe5",
+		"region.europe6",
+		"region.europe7",
+		"region.europe8",
+		"region.europe9",
+	}
+	platforms = []string{
+		"platform.pc",
+		"platform.xbox",
+		"platform.ps",
+		"platform.nintendo",
+		"platform.any",
+	}
+	playlists = []string{
+		"mmr.playlist1",
+		"mmr.playlist2",
+		"mmr.playlist3",
+		"mmr.playlist4",
+		"mmr.playlist5",
+		"mmr.playlist6",
+		"mmr.playlist7",
+		"mmr.playlist8",
+		"mmr.playlist9",
+		"mmr.playlist10",
+		"mmr.playlist11",
+		"mmr.playlist12",
+		"mmr.playlist13",
+		"mmr.playlist14",
+		"mmr.playlist15",
+	}
+)
+
+// PlaylistTicket generates a ticket for profile scale testing
+func PlaylistTicket() *pb.Ticket {
```

Comment:
Where does this get called? Ideally you want this to be configurable right? Also, considered the ticket to be the same for any scale test? Think like.. Just one ticket generation - but the different tests can use profiles that use different aspects of that Ticket?